examples/tuners/mnist_keras_customized_advisor/search_space.json

```diff
@@ -0,0 +1,4 @@
+{
+    "optimizer": ["Adam", "SGD"],
+    "learning_rate": [0.001, 0.1]
```

Comment:
it might be confusing that this search space does not follow the defined spec. It would be better to use the defined spec, or add explanation at a proper place.